art/attacks/poisoning/bad_det/bad_det_rma.py

```diff
@@ -93,28 +93,28 @@ def poison(  # pylint: disable=W0221
         :param y: True labels of type `List[Dict[np.ndarray]]`, one dictionary per input image.
                   The keys and values of the dictionary are:
                   - boxes [N, 4]: the boxes in [x1, y1, x2, y2] format, with 0 <= x1 < x2 <= W and 0 <= y1 < y2 <= H.
-                  - labels [N]: the labels for each image
-                  - scores [N]: the scores or each prediction.
```

Comment:
The `scores` field is not required for this attack since only `labels` and `boxes` are used. But for consistency, this can be re-added to the docstring.